waku/v2/protocol/pb/codec.go

```diff
@@ -0,0 +1,29 @@
+package pb
+
+import (
+	"google.golang.org/protobuf/encoding/protojson"
+	"google.golang.org/protobuf/proto"
+)
+
+func (m *WakuMessage) MarshalJSON() ([]byte, error) {
+	return (protojson.MarshalOptions{}).Marshal(m)
```

Comment:
The issue noticed was here https://github.com/waku-org/go-waku/blob/43412c9da5927d16c8befcb19d1511e707a9ef70/waku/v2/protocol/pb/message.pb.go#L31

where `content_topic` was being picked instead of `contentTopic` during decode.
How did both got assigned to the field? Was there some manual edit to the pb.go file post generation?